src/sass/components/button.scss

```diff
@@ -82,17 +82,124 @@
   }
 }
 
+@mixin alt-tetra-button($button){
```

Comment:
Is it necessary and relevant to create a new function whereas `tetra-button()` is very similar?
You are able to yield as a second argument a boolean `IsAlt`.

Example:
@mixin tetra-button($button, isAlt=false)
@include tetra-button(default) +> Default button
@include tetra-button(default, true) +> Alt default button

The refactoring looks simple. 
Please Have a try.